app/commands/git/temp/import_pull_requests_for_repo.rb

```diff
@@ -0,0 +1,140 @@
+module Git
+  module Temp
+    class ImportPullRequestsForRepo
+      include Mandate
+
+      initialize_with :repo
+
+      def call
+        fetch!
+        create!
+      end
+
+      private
+      attr_reader :pull_requests
+
+      def create!
+        pull_requests.each do |pr|
+          ::Git::PullRequest.create!(
+            node_id: pr[:pr_id],
+            number: pr[:pr_number],
+            author: pr[:author],
+            repo: pr[:repo],
+            event: pr
+          )
+        rescue ActiveRecord::RecordNotUnique
+          nil
+        end
+      end
+
+      def fetch!
+        cursor = nil
+        @pull_requests = []
+
+        loop do
+          page_data = fetch_page(cursor)
+
+          # TODO: filter out PRs we want to ignore (e.g. the v3 bulk rename PRs)
+          @pull_requests += pull_requests_from_page_data(page_data)
+
+          page_info = page_data[:data][:repository][:pullRequests][:pageInfo]
+          cursor = page_info[:endCursor]
+          break unless page_info[:hasNextPage]
+
+          # If the rate limit was exceeded, sleep until it resets
+          rate_limit = page_data[:data][:rateLimit]
+          next unless rate_limit[:remaining] <= 0
+
+          reset_at = Time.parse(rate_limit[:resetAt]).utc
+          seconds_until_reset = reset_at - Time.now.utc
+          sleep(seconds_until_reset.ceil)
+        end
+      end
+
+      def fetch_page(cursor)
+        query = "{
+          repository(owner: \"#{repo_owner}\", name: \"#{repo_name}\") {
+            nameWithOwner
+            pullRequests(first: 100, states:[CLOSED, MERGED] #{", after: \"#{cursor}\"" if cursor}) {
+              nodes {
+                id
+                url
+                title
+                merged
+                number
+                labels(first: 100) {
+                  nodes {
+                    name
+                  }
+                }
+                author {
+                  login
+                }
+                reviews(first: 100) {
+                  nodes {
+                    author {
+                      login
+                    }
+                  }
+                }
+              }
+              pageInfo {
+                hasNextPage
+                endCursor
+              }
+            }
+          }
+          rateLimit {
+            remaining
+            resetAt
+          }
+        }".freeze
+
+        octokit_client.post("https://api.github.com/graphql", { query: query }.to_json)
```

Comment:
Side note: the octokit.rb library has _no_ support for the graphql API. I've looked at other graphql libaries, but they don't seem to offer much value, so I opted for the simple option which is to use the octokit client to do a POST request with the graphql query.